deployments/employment_outcomes.yaml

```diff
@@ -0,0 +1,58 @@
+url_slug: post-secondary-employment-outcomes-u-s-census-bureau-2019
+status: Draft
+registry_authors:
+  - Priyanka Nanayakkara
+tier: 2
+deployment:
+  name: Post-Secondary Employment Outcomes
+  data_curators:
+    - U.S. Census Bureau
+  intended_use: For college students and prospective college students to be informed about relationships between specific fields of study and earning outcomes.
+  data_product_type: Summary statistics
+  data_product_region: United States
+  description: Summary statistics that describe earning outcomes for college graduates at different levels, including but not limited to, degree level, degree field, and state of institution.
+  publication_date: '2019-01-01'
+  data_product_sector: Education
+
+  dp_flavor:
+    name: Pure DP
+    # input_metric: ''
+    # bound_on_input_distance: ''
+    # output_measure: ''
+    # bound_on_output_distance: ''
+    # data_domain:
+
+    # unprotected_quantities:
+
+  privacy_loss:
+    privacy_unit: individual-level
```

Comment:
Maybe capitalize to Individual-level.

I realize the Census doesn't have "users" but should think if "Individual-level" is meaningfully different from "User-level" and if so (or if not), what that means for the schema.

Reply:
- Filed #115: Not sure if enum or regex makes more sense.